src/Command/GenerateJsonCommand.php

```diff
@@ -107,6 +107,8 @@ private function generatePrestaShopModulesJson(
         OutputInterface $output
     ): void {
         $infos = [];
+        $devVersion = new PrestaShop('9.0.0');
+        $prestashopVersions[] = $devVersion;
```

Comment:
```suggestion
        // Add current development version so that its module's list is available, but only if it is not part of the released ones
        $developmentVersion = '9.0.0';
        $addDevelopmentVersion = true;
        foreach ($prestashopVersions as $prestashopVersion) {
            if ($prestashopVersion-> getVersion() === $developmentVersion) {
                $addDevelopmentVersion = false;
            }
        }
        if (addDevelopmentVersion) {
            $prestashopVersions[] = new PrestaShop(developmentVersion);
        }

```